vm/actor/src/builtin/miner/state.rs

```diff
@@ -434,8 +438,8 @@ impl State {
         F: FnMut(ChainEpoch, &BitField) -> Result<(), String>,
     {
         let sector_arr = Amt::<BitField, _>::load(&self.fault_epochs, store)?;
-
-        sector_arr.for_each(|i, v| f(i, v))
+        // TODO AMT and Node for_each should be i64
```

Comment:
I don't think that's the best way for this, the index is meant to index an "array" so it being signed is kinda wrong